src/code/reducers/helpers/load-state-from-authoring.js

```diff
@@ -105,10 +105,24 @@ export function loadStateFromAuthoring(state, authoring, progress={}) {
         template = templateName && templates[templateName];
   if (!template) return state;
 
+  let userChangeableGenes = authoredChallenge.userChangeableGenes;
```

Comment:
This was the only tricky bit - until, the new `userChangeableGenes` format didn't have its authored gene string split into an array, like the old format did. Now the behavior matches for both formats. I got kind of lucky that this even worked before...